### PR TITLE
fix: merge gerrit dev branch

### PR DESCRIPTION
**Pull request by myk1343**
merge gerrit dev branch

Log: merge gerrit dev branch

### CODE REVIEW

File: src/src/ui/CMakeLists.txt [hunk 1]
```diff
@@ -129,9 +129,9 @@ target_link_libraries(dlmcontrolui
     dlmlog
     dlmaria2
     dlmdatabase
-    Dtk::Core
-    Dtk::Widget
-    Dtk::Gui
+    ${DtkCore_LIBRARIES}
+    ${DtkWidget_LIBRARIES}
+    ${DtkGui_LIBRARIES}
```
[the remark this answers is not in the document]
Comment (myk1343) [reply src/src/ui/CMakeLists.txt:134]: v20上编译不过

File: debian/source/format [hunk 2]
```diff
@@ -1 +1 @@
-3.0 (native)
+3.0 (quilt)
```
[the remark this answers is not in the document]
Comment (myk1343) [reply debian/source/format]: 已改回

**Comment by hundundadi** [comment]
[是否满足兼容性要求] Y
[是否满足commit提交规范] Y
[是否满足编码规范] Y
[Review结论] Pass
[Fail原因] N/A

**Comment by pengfeixx** [comment]
[是否满足兼容性要求] Y
[是否满足commit提交规范] Y
[是否满足编码规范] Y
[Review结论] Pass
[Fail原因] N/A

**Review by lzwind — APPROVED** [review]
/+1

**Pull request merged**